### PR TITLE
fix(Tag/native): update native tag wrt new guidelines

### PR DESCRIPTION
**Pull request by kashishgrover**

### CODE REVIEW

**Comment by kamleshchandnani** [comment]
Deploy preview for *leaf-ui* processing.

Built with commit 171d5136d41b8dc12944601fc0137621265ada48

https://app.netlify.com/sites/leaf-ui/deploys/5ad5d96e11b73b46d1add8d4

File: src/components/Tag/native/Tag.js [hunk 1]
```diff
@@ -3,37 +3,41 @@ import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import Text from '../../Text/native';
 
+const styles = {
+  background(props) {
+    if (props.kind === 'filled') return props.theme.color[props.color];
+    return props.theme.color[`${props.color}Lighter`];
+  },
+  padding(props) {
+    if (props.size === 'small') return props.theme.px([0.15, 0.5, 0.25, 0.5]);
```
Comment (srinivasdamam) [inline src/components/Tag/native/Tag.js]: Vertical padding not consistent ?

Comment (lakshyaranganath) [reply src/components/Tag/native/Tag.js]: Good catch! 

@kashishgrover could you look into this and make sure it's as per our guidelines?

File: src/components/Tag/native/Tag.js [hunk 2]
```diff
@@ -3,37 +3,41 @@ import PropTypes from 'prop-types';
...
-    `,
-  }[p.kind])}
-  border-radius: 100px;
-  border-width: 1px;
-  border-color: ${(p) => p.theme.color[p.color]};
-  `;
+  border-radius: ${styles.borderRadius};
+  padding: ${styles.padding};
+  background: ${styles.background};
+  border-width: 0.8;
```
Comment (lakshyaranganath) [inline src/components/Tag/native/Tag.js]: All border-width's need to be 1px

File: src/components/Tag/native/Tag.js [hunk 3]
```diff
@@ -51,20 +55,18 @@ const Tag = ({ style, color, kind, size, children }) => (
 );
 
 Tag.propTypes = {
```
Comment (lakshyaranganath) [inline src/components/Tag/native/Tag.js]: Make sure, propTypes and defaultProps are as per guidelines too. Tag's need to accept the shape `sharpEdged` as well. Take a look at the web version of the Tag component for the props.

**Pull request merged**